build/tools/esp32/esptool_py/romfs_flash_address.sh

```diff
@@ -31,11 +31,11 @@ i=0
 #find romfs partition number
 for pname in $flash_type_list
 do
-	if [ "$pname" == "romfs" ]; then
```

Comment:
LGTM，

and how about we explicitly call 'bash' in this file  by '#!/bin/bash'?

Reply:
Current modification will both works for "bash" and "dash". 
if explicitly specify "bash", how about it is not exist in some machine?